jscomp/gentype_tests/typescript-react-example/src/ImportHookDefault.res.js

```diff
@@ -4,11 +4,11 @@ import ImportHookDefaultGen from "./ImportHookDefault.gen";
 import * as ImportHookDefaultGen$1 from "./ImportHookDefault.gen";
 
 function make(prim) {
-  return ImportHookDefaultGen$1.make;
+  return ImportHookDefaultGen$1.make(prim);
```

Comment:
This change in the output looks incorrect to me.
Before it returned the make function, now it applies it and returns the result.

Reply:
Or was it broken before?

Reply:
It was broken before.

Reply:
Ah yes, I missed this in the review of the previous PR. 🙈